tests/SchemaTest.php

```diff
@@ -397,6 +399,13 @@ public function testGetTables()
         DB::connection('mongodb')->table('newcollection_two')->insert(['test' => 'value']);
         $dbName = DB::connection('mongodb')->getDatabaseName();
 
+        // Create a view (this creates system.views)
+        DB::connection('mongodb')->getDatabase()->command([
+            'create' => 'test_view',
+            'viewOn' => 'newcollection',
+            'pipeline' => [],
+        ]);
```

Comment:
That's a good idea to create a view to initialize a system collection. You could also create a GridFS bucket.

You can use the `createCollection` method instead of the raw command. The pipeline is optional.
```suggestion
        DB::connection('mongodb')->getDatabase()->createCollection('test_view', ['viewOn' => 'newcollection']);
```